### DISCUSSION TITLE
Ability to add time-log to daily journal?

### DISCUSSION THREAD

**Discussion by MurielCuiCui** [TriliumNext/Trilium#6347] [Q&A] [answered]
hi, I want to add a time-log table into daily journal , it is easy to create one, but it is cumbersome to do everyday . It will be convenience if time-log be added in java-script frontend . I don't know how to build it. Someone can help me ?

**Comment by zadam** [comment]
It would help if you'd clarify what exactly time-log means to you ...

**Comment by MurielCuiCui** [comment]
I want to log what I have done every hour with the table like this.
![image](https://user-images.githubusercontent.com/37406633/57191813-3ba73900-6f5c-11e9-8747-b009442bc3f7.png)

**Comment by MurielCuiCui** [comment]
I don't know much about javascript, so it's a little difficult for me to create a function which inserts a table into texts.

**Comment by eliandoran** [comment, marked as answer]
This seems relatively easily achievable by simply defining a template for the day notes, with the desired table in it.
